src/Quibill.Domain/Models/SingleTransaction.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Quibill.Domain
+{
+    class SingleTransaction : ITransaction
+    {
+        int ITransaction.TransactionId { get; }
```

Comment:
The way interfaces are implemented is a little strange. basically you would just use 'public int TransactionId { get; }' and similar for the rest of them. you can leave off the ITransaction part, but you will need to add the public keyword.